fmn/consumer/__init__.py

```diff
@@ -1,3 +1,3 @@
-from consumer import FMNConsumer
-from producer import ConfirmationProducer
-from producer import DigestProducer
+from consumer import FMNConsumer  # noqa
```

Comment:
I've set this up to run with flake8 - I'm not sure what other tools honor this, but it tells flake8 to ignore this particular line because it's violating PEP8 for good reason. In this case it's importing classes without using them in the module so that they're available to users who do something like ``from fmn import consumer`` and then want access to everything in the consumer package.